lua/entities/gmod_wire_expression2/core/custom/cfc_entity.lua

```diff
@@ -52,4 +52,21 @@ e2function void entity:ejectPodTo(vector pos)
 
     driver:ExitVehicle()
     driver:SetPos(clampedPos)
-end
\ No newline at end of file
+end
+
+
+-- Easy way to have some of the setPos benefits without mostly being able to be abused.
+e2function number entity:setFairPos(vector pos)
+    if this:IsPlayer() then return 3 end -- Debug output 3 if the target is a player, because we dont want someone tping themselves around the e2 or any other player.
```

Comment:
Typically you will simply return `false` here. If you needed the user to know what went wrong you could add a ChatPrint in here. Numeric return codes aren't useful here.

Comment is unnecessary as it's self explanatory.

Reply:
i see an use case of people using the debug codes for other things as well.

Reply:
Removed comment

Reply:
There may be such a case, but this is an antipattern for E2. Very few, if any other functions do such a thing.

Reply:
Could i make it a client convar or something

Reply:
I agree with you, no other functions do debug like this, and so i will remove them.